eqcorrscan/core/match_filter/helpers.py

```diff
@@ -62,7 +61,10 @@ def _spike_test(stream, percent=0.99, multiplier=1e7):
                'FFT prior to cross-correlation.\n' +
                'If you think this spike is real please report ' +
                'this as a bug.')
-        raise MatchFilterError(msg)
+        print(msg)
+        for ID in list_ids:
+            stream.remove(stream.select(id=ID)[0])
```

Comment:
The reason I *don't* do this is because it removes a lot of data for potentially just one spiked data point. This seems like "throwing the baby out with the bathwater". I leave it up to the user to work out what to do about the spikes found by this function. In general removing the whole trace seems like the worst option. If you want to pursue this I would recommend just cutting-out a region in the trace around the spike. The pre-processing functions cope with gaps in data well so this could be a good option.